GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3058,6 +3058,10 @@ private void manageWorktreeToolStripMenuItem_Click(object sender, EventArgs e)
         {
             using FormManageWorktree formManageWorktree = new(UICommands);
             formManageWorktree.ShowDialog(this);
+            if (formManageWorktree.ShouldRefreshRevisionGrid)
+            {
+                RefreshRevisions();
+            }
```

Comment:
We don't seem to be using the dialog result from `ShowDialog` call. Is there a way to use that instead of introducing a new public property?

Reply:
No, it seems not possible for me because the user will close with the close button (so with DialogResult.Cancel) but depending on what he did, the state of the flag will be different. So it means we will have to intercept the action and change the dialogResult value. At best it's a big hack and we will loose expressiveness, at worst, I don't know if it's really possible...